lib/player.js

```diff
@@ -5728,8 +5729,16 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       // Bad stream?
       return;
     }
-    const liveSyncMaxLatency = this.config_.streaming.liveSyncMaxLatency;
-    const liveSyncPlaybackRate = this.config_.streaming.liveSyncPlaybackRate;
+
+    // Should use serviceDescription if they exist.
+    const liveSyncMaxLatency =
```

Comment:
After looking at this carefully I think the logic should be:
- If the user has activated the livesync configuration, we must use the configuration values.
- If the user has not activated the configuration and we have defined this.manifest_.serviceDescription we must use the values defined in serviceDescription, and if there is no value, use the values of the configuration.

Reply:
I am not sure about this suggestion, let me explain.

This MPD has `<ServiceDescription />` but just `<Latency target="3000" referenceId="1"/>` is defined.

https://shaka-player-demo.appspot.com/demo/#audiolang=es-ES;textlang=es-ES;uilang=es-ES;asset=https://akamaibroadcasteruseast.akamaized.net/cmaf/live/657078/akasource/out.mpd;panel=ALL_CONTENT;build=uncompiled

In this case, we would be using streaming.liveSyncMaxLatency and streaming.liveSyncPlaybackRate as fallbacks.

Is this the expected behavior?

As an alternative I would suggest this flow:

- if liveSync is set, liveSyncMaxLatency and liveSyncPlaybackRate are used.
- if liveSync is not set and serviceDescription.maxLatency and serviceDescription.maxPlaybackRate are used

They differ in the fallback when serviceDescription.maxLatency and serviceDescription.maxPlaybackRate are not defined.

What's your opinion?

Reply:
if liveSync is true, liveSyncMaxLatency and liveSyncPlaybackRate are used.
If liveSync is false and serviceDescription has some property, we use serviceDescription.maxLatency and serviceDescription.maxPlaybackRate, and if some of one are missing, we use liveSyncMaxLatency and liveSyncPlaybackRate as fallback

Reply:
I have finally used your suggestion but checking `this.config_.streaming.liveSync` before using `this.config_.streaming.liveSyncMaxLatency` and `this.config_.streaming.liveSyncPlaybackRate` because we may reach this point with serviceDescription set but a false liveSync